VGHtmlParser/Classes/Public/VGHtmlParser.m

```diff
@@ -62,7 +62,13 @@ - (NSAttributedString *)parse
     NSArray *elements = [doc searchWithXPathQuery:@"/"];
     TFHppleElement * element = [elements firstObject];
     NSAttributedString *attrString = [self attrStringForElement:element];
-    return [self trimWhitespaceAndNewLine:attrString];
+    
+    NSString *string = [[NSString alloc] initWithData:self.htmlData encoding:NSUnicodeStringEncoding];
+    if ([string hasPrefix:@"<ul>"]) {
```

Comment:
this check seems pretty random & out of context, but on the other hand, lets hope that we can deprecate this library in future anyway :)

Reply:
Yes, but unfortunately I do not have better ideas, how to skip new line trimming at the beginning.

Talking about deprecation, we could replace it with [SwiftSoup](https://github.com/scinfu/SwiftSoup), which seems popular.

Should we?